heron/common/src/java/com/twitter/heron/common/utils/metrics/ComponentMetrics.java

```diff
@@ -13,16 +13,14 @@
 //  limitations under the License.
 package com.twitter.heron.common.utils.metrics;
 
-import com.twitter.heron.classification.InterfaceAudience;
-import com.twitter.heron.classification.InterfaceStability;
-
 /**
- * Interface for common metric actions that both spouts and bolts support
+ * Abstract Class for common metric actions that both spouts and bolts support
  */
-@InterfaceAudience.Private
-@InterfaceStability.Evolving
-public interface ComponentMetrics {
+public abstract class ComponentMetrics {
+  // Metric-name suffix reserved for value aggregating on all different streams
+  public static final String ALL_STREAMS_AGGREGATED = "__all-streams-aggregated";
```

Comment:
One the one hand, it seems we don't have such Constants class;
on the other hand, we have a class designed to abstract out the shared actions/implementations, which IMO is a good place to put some shared things, like a shared field.